django/core/management/commands/shell.py

```diff
@@ -54,27 +56,18 @@ def add_arguments(self, parser):
     def ipython(self, options):
         from IPython import start_ipython
 
-        start_ipython(
-            argv=[],
-            user_ns=self.get_and_report_namespace(
-                options["verbosity"], options["no_imports"]
-            ),
-        )
+        start_ipython(argv=[], user_ns=self.get_namespace(**options))
```

Comment:
We could pull out the using `**options` as a refactor commit (before or after the name change), including the similar test changes

Reply:
Makes sense, splitting.